### DISCUSSION TITLE
very long pane titles, help me fix it

### DISCUSSION THREAD

**Discussion by artempoo** [catppuccin/tmux#433] [Help]
Initially everything looks fine, I set my headers via the command:
`tmux select-pane -T ...`

![image](https://github.com/user-attachments/assets/dcf014bd-a39c-4d79-819c-3202e5cd2368)

However, as soon as I open a directory with a large nesting, for example in yazi or nvim, it becomes completely confusing and is of no use.

![image](https://github.com/user-attachments/assets/2c6ae6fb-9036-4722-8aa7-98ea2eea51f1)

How can I fix this please help?
I want the headers to always look like the first screenshot without updating

mac os vertuna
kitty, zsh, tmux

**Comment by kjnsn** [comment]
This has nothing to do with tmux or catppuccin. You have somehow configured neovim to set that title

**Reply by artempoo** [reply]
if I configured it for neovim , it would only work for neovim , is that logical?

But it works the same in yazi , it's a separate application from neovim , and I didn't configure yazi separately in any way.

![image](https://github.com/user-attachments/assets/e64a0c81-f437-44b7-aaad-ce7fa2f60524)

moreover, if you disable the catppuccin theme, the problem does not repeat itself, neither in yazi, nor in neovim

**Reply by kjnsn** [reply]
The problem _is_ repeating itself. Look on the right where the pane title is, yazi is setting it to something long but tmux is truncating it at 40 characters (because that is the default as documented)

**Reply by artempoo** [reply]
I understand you, thank you for your attention and help, I will dig in the direction of yazi and nvim
